src/tests/client.test.js

```diff
@@ -1,54 +1,44 @@
 import jest from 'jest-mock';
-
 import { make_client } from '../client.js';
 
 /**
- * Test the client
+ * Test the client without any server
  */
 
 describe('client', () => {
-
   it('should return results: mock', async () => {
-
-    // keep a copy of fetch
+    // Keep a copy of fetch
     const nodeFetch = fetch;
 
-    const mockResponse = {
-      body: {
-        results: [{
-          hits: [{
-            description: 'test',
-            id: 1,
-            name: 'test',
-            postgres_searchbox_v1_doc: "'coalesc':1,3 'descript':4 'name':2",
-          },],
-        },], query: 'test',
-      }
+    try {
+      const mockResponse = {
+        body: {
+          results: [{ hits: [{ description: 'test', id: 1, name: 'test' }] }],
+          query: 'test',
+        },
+      };
+
+      global.fetch = jest.fn(() =>
+        Promise.resolve({
+          ok: true,
+          json: () => Promise.resolve(mockResponse),
+        })
+      );
+
+      const client = make_client('http://localhost:3002/api/search');
```

Comment:
I thought this would be changed https://github.com/dekimir/postgres-searchbox/pull/3#discussion_r1166121080